backend/layers/api/portal_api.py

```diff
@@ -79,11 +79,11 @@ def get_collections_list(from_date: int = None, to_date: int = None, token_info:
             "owner": c.owner,
             "created_at": c.created_at,
         }
-        if c.published_at is not None or c.canonical_collection.originally_published_at is None:
-            collection["id"] = c.collection_id.id
-        else:
+        if c.is_unpublished_version():
```

Comment:
Is `c` is a collection or a collection version? If a Collection, it sounds odd to have a method on it with "version" in the name.    If a CollectionVersion, I would change var naming to indicate "version"